docs/resources/tunnel_config.md

```diff
@@ -139,3 +139,11 @@ Optional:
 Optional:
 
 - `enabled` (Boolean) Whether WARP routing is enabled.
+
```

Comment:
this file is automatically generated using the process outlined at https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/contributing/registry-documentation.md. for this to persist, you'll need to update the example import file otherwise it will be overwritten on generation.

Reply:
i went ahead and added these in 542bfc8de156c4be2b0df7186ebebbb8491cc1f0